src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Color color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(101);
```

Comment:
same for Random here
make 101 a constant

Reply:
I made number 101 constant but how i can move this `int number = new Random().nextInt(101);` to the class level?

Reply:
I guess maybe i fixed it